app/src/main/java/com/example/bigowlapp/model/Attendance.java

```diff
@@ -0,0 +1,67 @@
+package com.example.bigowlapp.model;
+
+import com.google.firebase.firestore.Exclude;
+
+public class Attendance {
+    public enum LocatedStatus {
+        NOT_DETECTED,
+        CORRECT_LOCATION,
+        WRONG_LOCATION
+    }
+
+    private boolean authenticated = false;
+    private boolean authAttemptedMethod1 = false;  // Authentication with first method
+    private boolean authAttemptedMethod2 = false;  // Authentication with second method
```

Comment:
Provide better naming and maybe briefly comment what they do

Reply:
Since these variables are actively being worked with on a different branch, I'll leave it as is for now.

Although I agree with your point.